didx-xyz/yoma-mobile
# YOMA-446 - feat(CompletedCourses): adding all CV components

## Scope [YOMA-446](https://yomaworld.atlassian.net/browse/YOMA-446)
adding the CV components (Add form, List view and widget) for Completed Courses.

![](https://media.giphy.com/media/xsMJyTdf4BQKk/giphy.gif)
## Work Done


## Steps to Test
- open MyCv - see Completed Courses widget. Click the + to add a new course, click the 'view all' to view the current courses.
    
## Screenshots
![Screenshot 2022-06-20 at 21 57 28](https://user-images.githubusercontent.com/1424561/174670311-e39c0976-5295-47d9-aa57-264a8ba43fca.png)
![Screenshot 2022-06-20 at 21 57 39](https://user-images.githubusercontent.com/1424561/174670322-8f6d7ff9-f3b8-4388-a19e-706bf8d266ae.png)
![Screenshot 2022-06-20 at 21 57 50](https://user-images.githubusercontent.com/1424561/174670327-39ab66d1-8a65-4fd9-9527-4186b481cfac.png)
